static/js/emoji_picker.js

```diff
@@ -19,6 +19,22 @@ var search_results = [];
 var section_head_offsets = [];
 
 function get_emoji_categories() {
```

Comment:
I initially considered the choice of naming it `get_all_emoji_categories` vs `default_emoji_categories`.
I guess we can rename to the former just fine.